securedrop/management/sources.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+from typing import List
+
+from db import db
+from encryption import EncryptionManager, GpgKeyNotFoundError
+from management import app_context
+from models import Source
+
+
+def remove_pending_sources(args: argparse.Namespace) -> int:
+    """
+    Removes pending source accounts, with the option of keeping
+    the n newest source accounts.
+    """
+    n = args.keep_most_recent
```

Comment:
nit: I think we should avoid one-character variables whenever possible, in this case why not pass it directly, `find_pending_sources(args.keep_most_recent)`